src/main/kotlin/io/snyk/plugin/ui/toolwindow/LabelProvider.kt

```diff
@@ -49,6 +49,8 @@ class LabelProvider {
 
     fun createLinkLabel(url: URL, text: String): LinkLabel<*> {
         val openLinkAction = OpenLinkAction(url)
-        return LinkLabel.create(text, openLinkAction)
+        return LinkLabel.create(text, openLinkAction).apply {
```

Comment:
This is not correct - the toolTipText should be provided as a parameter, as the link could be something else  than a description.